### ISSUE TITLE
File limit full IT is unstable for Windows Agent in 4.4

### ISSUE THREAD

**Issue by Rebits** [wazuh/wazuh-qa#4148]
### Nightly 

| **Date** | **Commit** | 
| --- | --- |
| 2023-05-04 | https://github.com/wazuh/wazuh/commit/94916de |

### Description

The `test_file_limit_full` aims to validate the functionality of the 'wazuh-syscheckd' daemon in generating appropriate events when the FIM (File Integrity Monitoring) database is in 'full database alert' mode and reaches the limit of files to monitor set in the 'file_limit' tag. To accomplish this, the following steps will be executed:

- Set up a directory specifically for testing purposes.
- Create multiple testing files within the monitored directory until the file monitoring limit is reached.
- Check if the FIM event labeled as 'full' is generated each time a new testing file is added to the monitored directory.
- Verify that the number of inodes and monitored files matches in the FIM event.

By performing this test, we can ensure that the 'wazuh-syscheckd' daemon behaves correctly and generates the expected events when the file monitoring limit is reached, specifically indicating the 'full' status in the FIM event, while ensuring the accuracy of the inodes and monitored files count.

However, in the failed build, it seems that the DB full alert event is never triggered. In the agent logs we can check that the database deletion failed. In addition, agent lost connection with the manager.

```
2023/05/04 04:09:55 wazuh-agent[3984] fim_db.c:155 at fim_db_clean(): DEBUG: (6340): Failed to delete FIM database 'queue/fim/db/fim.db'- 1Âº try.
2023/05/04 04:09:55 wazuh-agent[3984] wait_op.c:74 at os_wait_primitive(): DEBUG: Process locked due to agent is offline. Waiting for connection...
2023/05/04 04:10:00 wazuh-agent[3984] state.c:78 at write_state(): DEBUG: Updating state file.
```

### Evidences 

- [FIM_Windows_Report.zip](https://github.com/wazuh/wazuh-qa/files/11394915/agent_windows_html_report_Test_integration_B38097_20230504044645.zip)

**Comment by roronoasins** [comment]
During https://github.com/wazuh/wazuh/issues/17739 it was reported a similar error within the report. It is required to fix it also in 4.6

**Comment by vikman90** [comment]
Closing this issue. Superseded by version 5.0.
